packages/react-input/Spec.md

```diff
@@ -1,63 +1,322 @@
+| onChange                     | `(ev, value) => void`                                 | `(ev, data: { ...props, value }) => void`                   | `(ev, data: { value: string }) => void` |
+| container className          | `className?: string`                                  |                                                             | top-level `className`                   |
+| input className              | `inputClassName?: string`                             | `input.className?: `                                        |                                         |
+| aria label                   | `ariaLabel?: string`                                  | `'aria-label'?: string`                                     | `'aria-label'?: string`                 |
+| clearable                    | n/a                                                   | `clearable?: boolean`                                       | TBD (deferred)                          |
+| theme                        | `theme?: ITheme`                                      |                                                             | handled by context                      |
+| style overrides              | `styles?: IStyleFunctionOrObject<...>`                | `styles?: ComponentSlotStyles<...>`                         | className, slot classNames              |
+| accessibility                | individual props                                      | `Accessibility<InputBehaviorProps>` (see below)             | individual props?                       |
+
+```ts
+// packages/fluentui/react-northstar/src/utils/htmlPropsUtils.tsx
+type SupportedIntrinsicInputProps = {
+  [K in HtmlInputProps]?: K extends keyof JSX.IntrinsicElements['input'] ? JSX.IntrinsicElements['input'][K] : any;
+};
+// HtmlInputProps: a subset of React and HTML native props
+
+// packages/fluentui/react-northstar/src/components/Box/Box.tsx
+interface BoxProps extends UIComponentProps<BoxProps>, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+
+// packages/fluentui/accessibility/src/behaviors/Input/inputBehavior.ts
+type InputBehaviorProps = { disabled?: boolean; required?: boolean; error?: boolean };
+```
+
+Native props following standard behavior in both libraries + converged:
+
+- `disabled?: boolean`
+- `readOnly?: boolean` -- VERIFY v0
+- `autoComplete?: string` -- VERIFY v0
+- and any other native props not specified
+
+### Imperative API
+
+In v8 there's an explicit imperative API, which is accessed via `componentRef` following the [`ITextField` interface](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/TextField/TextField.types.ts#L9). The methods are used for getting the value and manipulating focus and selection.
+
+In v0, all these things can be done by calling methods on the `<input>` element itself, exposed via the top-level `ref`.
```

Comment:
Should we call out here that we are going to do something similar to what is being done in v0 for v9?